jwst/associations/lib/acid.py

```diff
@@ -17,7 +17,7 @@ class ACID():
 
     Parameters
     ----------
-    in: str or 2-tuple
+    in : str or 2-tuple
```

Comment:
This is probably OK, since it's just documentation, but `in` is a reserved word in Python.

Reply:
That should have been `input`. Fixed.